packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -118,19 +101,30 @@ contract OPContractsManager is ISemver, Initializable {
         address permissionedDisputeGame2;
     }
 
+    /// @notice The latest implementation contracts for the OP Stack.
+    struct ImplementationContracts {
```

Comment:
nit since we know these are contracts and it makes this consistent with `Blueprints` naming convention (as opposed to `BlueprintContracts`)
```suggestion
    struct Implementations {
```

Reply:
Change made here: https://github.com/ethereum-optimism/optimism/pull/12851/commits/c8068686416d6eda365a44d2bd74c9b33ac736d7#diff-32c676c7a2cc25dc934a436546eccc2c3484b44491cc1569dce382ffb13b84c7R105